frontend/src/__mocks__/mockPipelineVersionsProxy.ts

```diff
@@ -1,13 +1,23 @@
 /* eslint-disable camelcase */
+import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
 import {
   ArtifactType,
   InputDefinitionParameterType,
+  PipelineKFCallCommon,
   PipelineVersionKF,
   PipelineVersionKFv2,
   RelationshipKF,
   ResourceTypeKF,
 } from '~/concepts/pipelines/kfTypes';
 
+export type ArgoWorkflowVersion = Omit<PipelineVersionKFv2, 'pipeline_spec'> & {
+  pipeline_spec: K8sResourceCommon;
+};
+
+export type BuildMockPipelinveVersionsType = PipelineKFCallCommon<{
+  pipeline_versions: (PipelineVersionKFv2 | ArgoWorkflowVersion)[];
+}>;
+
```

Comment:
Need to create these types to accommodate argo workflow. Is it fine to add it here, as these are only required in mocks and tests?

Reply:
`ArgoWorkflowVersion` can exists in the types for pipelines